repo_sync/cli.py

```diff
@@ -22,7 +22,8 @@
               help="Client address to synchronize.", required=True)
 @click.option('--file', '-f', 'file_path', type=click.Path(exists=True), prompt=True,
               help="File path to synchronize.", required=True)
-def sync_cli(address: List[str], file_path: str) -> None:
+@click.option('--server-chose', '-s', is_flag=True, help="chose server", required=True)
```

Comment:
requiredって指定しなくても良いような気がするのですがどうでしょう？

```suggestion
@click.option('--server-chose', '-s', is_flag=True, help="chose server")
```

Reply:
確かに、必須ではないのでいらないですね。